editors/vim/Portfile

```diff
@@ -138,51 +138,81 @@ foreach s ${pythons_suffixes} {
     "
 }
 
-variant ruby requires ruby18 description {Compatibility variant, requires +ruby18} {}
-variant ruby18 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby30 description {Enable Ruby scripting} {
+variant ruby requires ruby33 description {Compatibility variant, requires +ruby33} {}
+variant ruby19 conflicts ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby26 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby20 conflicts ruby19 ruby21 ruby22 ruby23 ruby24 ruby25 ruby26 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby21 conflicts ruby19 ruby20 ruby22 ruby23 ruby24 ruby25 ruby26 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby22 conflicts ruby19 ruby20 ruby21 ruby23 ruby24 ruby25 ruby26 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby23 conflicts ruby19 ruby20 ruby21 ruby22 ruby24 ruby25 ruby26 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby24 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby25 ruby26 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby25 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby26 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby26 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby27 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby27 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby26 ruby30 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby30 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby26 ruby27 ruby31 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby31 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby26 ruby27 ruby30 ruby32 ruby33 description {Enable Ruby scripting} {}
+variant ruby32 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby26 ruby27 ruby30 ruby31 ruby33 description {Enable Ruby scripting} {}
+variant ruby33 conflicts ruby19 ruby20 ruby21 ruby22 ruby23 ruby24 ruby25 ruby26 ruby27 ruby30 ruby31 ruby32 description {Enable Ruby scripting} {}
+
+if {[variant_isset ruby33]} {
```

Comment:
Why do you use `variant_isset` instead of putting it into the variant body as it has been before?

Reply:
Separate version selection conflicts, and use `variant_isset` for logical separation. Keeping scripts clean.